conf/php-fpm.conf

```diff
@@ -347,16 +368,25 @@ chdir = __FINALPATH__
 ; Note: on highloaded environement, this can cause some delay in the page
 ; process time (several ms).
 ; Default Value: no
-catch_workers_output = yes
```

Comment:
And that?

Reply:
That's the default config, just applied it.